### ISSUE TITLE
Required Google Cloud SDK version does not support gcloud terraform vet

### ISSUE THREAD

**Issue by felipecrescencio-cit** [terraform-google-modules/terraform-example-foundation#766]
### TL;DR

Terraform-validator does not work with the Google Cloud SDK version 319.0.0 mentioned in `0-bootstrap` folder README.

### Expected behavior

Run step [5 commands](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/0-bootstrap#deploying-with-cloud-build) correctly.

> 5. To run terraform-validator steps please follow the [instructions](https://cloud.google.com/docs/terraform/policy-validation/validate-policies#install) to install the terraform-tools component.
    1. Run `terraform plan -input=false -out bootstrap.tfplan`
    1. Run `terraform show -json bootstrap.tfplan > bootstrap.json`
    1. Run `gcloud beta terraform vet bootstrap.json --policy-library="../policy-library" --project <A-VALID-PROJECT-ID>` and check for violations (`<A-VALID-PROJECT-ID>` must be an existing project you have access to, this is necessary because Terraform-validator needs to link resources to a valid Google Cloud Platform project).

### Observed behavior

Running `gcloud beta terraform vet` command I get the error below:
```bash
$ gcloud beta terraform vet bootstrap.json --policy-library="../policy-library" --project MY-PROJECT
ERROR: (gcloud.beta) Invalid choice: 'terraform'.
```
Trying to install terraform-tools with gcloud 319.0.0 I get the error below:
```bash
$ gcloud components install terraform-tools
ERROR: (gcloud.components.install) The following components are not available for your current SDK version [terraform-tools]. Please run `gcloud components update` to update your SDK.
```

### Terraform Configuration

```hcl
0-bootstrap folder configuration.
```

### Terraform Version

```sh
Terraform v0.13.7

gcloud version
Google Cloud SDK 319.0.0
beta 2020.11.13
```

### Additional information

Update Google Cloud SDK version mentioned in `0-bootstrap` folder README to [391.0.0](https://cloud.google.com/sdk/docs/release-notes#39100_2022-06-22).

This version enables `terraform-tools`.
